packages/itwin/tree-widget/package.json

```diff
@@ -81,6 +83,8 @@
     "@itwin/components-react": "^3.0.0",
     "@itwin/core-frontend": "^3.0.0",
     "@itwin/core-react": "^3.0.0",
+    "@itwin/itwinui-icons-react": "^1.8.0",
```

Comment:
the icon pkgs should not be peer deps, they should just be direct deps
i also dont agree with itwinui-react be listed as dev+peer